docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -428,7 +400,16 @@ endif::[]
 EOF
 ----
 
-* Optionally enable neutron-sriov-nic-agent in the OpenStackDataPlaneNodeSet CR
+. Make sure that ovn-controller settings configured in the `OpenStackDataPlaneNodeSet` CR are the same as were set in the Compute nodes before adoption. This configuration is stored in the "external_ids" column in the "Open_vSwitch" table in ovsdb:
```

Comment:
Change `make sure` to `ensure`:
https://source.redhat.com/groups/public/openstack-documentation-team/openstack_documentation_wiki/rhosp_documentation_style_checklist#jive_content_id_Phrasal_verbs